docs/index.rst

```diff
@@ -28,6 +26,30 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available Solidity Integrations
+-------------
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.	
+
+* `Ethereum Studio <https://live.ether.camp/>`_
+	A fully-functional well-designed WEB IDE for Ethereum Solidity developers.
+
+* `Visual Studio Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
+	Consensys integration of Solidity into Microsoft Visual Studio 2015
```

Comment:
I think it is not too relevant who made this.

`Solidity plugin for Microsoft Visual Studio that includes the Solidity compiler.`